OpenVote/src/com/openvote/Key.java

```diff
@@ -0,0 +1,16 @@
+package com.openvote;
+
+import java.util.UUID;
+
+public class Key {
+
+	UUID voteKey;
+
+	private UUID getVoteKey() {
+		return voteKey;
+	}
+
+	private void setVoteKey(UUID voteKey) {
+		this.voteKey = voteKey;
```

Comment:
Okay, so the KeyGenerator (my part) will have the single UUID for our project. It will instantiate one of your VoteKeys and pass in a new random UUID. We can do the setting in a VoteKey constructor, and then we don't need a set method, because the key should never change UUID.